src/main.rs

```diff
@@ -111,6 +111,13 @@ fn create_app<'a, 'b>(
                 .takes_value(true)
                 .group("vm-config"),
         )
+        .arg(
+            Arg::with_name("initramfs")
+                .long("initramfs")
+                .help("Path to initramfs image")
+                .takes_value(true)
+                .group("vm-config"),
+        )
         .arg(
```

Comment:
We also need to add this to our REST API by editing `vmm/src/api/openapi/cloud-hypervisor.yaml` and adding an `initramfs` property to the `VmConfig` scheme.

Reply:
is just adding the paramter to the file enough? I did just that.